src/app/admin/moderation/apologies/page.tsx

```
@@ -0,0 +1,127 @@
+                        </thead>
+                        <tbody className="bg-white dark:bg-gray-800 divide-y divide-gray-200 dark:divide-gray-700">
+                            {apologies.map((apology) => (
+                                <tr key={apology.id}>
+                                    <td className="px-6 py-4 whitespace-nowrap">
+                                        <div className="text-sm font-medium text-gray-900 dark:text-white">{apology.user_name}</div>
+                                        <div className="text-sm text-gray-500 dark:text-gray-400">{apology.user_email}</div>
+                                    </td>
+                                    <td className="px-6 py-4 whitespace-nowrap">
+                                        <div className="text-sm text-gray-900 dark:text-white">
+                                            {apology.llm_evaluation_score ? (apology.llm_evaluation_score * 100).toFixed(0) + '%' : 'N/A'}
+                                        </div>
```

ESLint

[error] Replace `······················{apology.llm_evaluation_score·?·(apology.llm_evaluation_score·*·100).toFixed(0)·+·'%'·:·'N/A'}⏎······················` with `{apology.llm_evaluation_score⏎························?·(apology.llm_evaluation_score·*·100).toFixed(0)·+·"%"⏎························:·"N/A"}⏎··`

(prettier/prettier)